Godeps/_workspace/src/k8s.io/kubernetes/pkg/master/master.go

```diff
@@ -168,6 +168,11 @@ type Config struct {
 	SSHUser       string
 	SSHKeyfile    string
 	InstallSSHKey InstallSSHKey
+
+	// Used to make the service error resolver aware of apis that may be installed in addition
+	// to the kubernetes api so it can parse request info appropriately and show more detailed
+	// error messages.
+	ExtendedApiPrefixes []string
```

Comment:
they have an experimental flag for the Kube api enablement but what I needed was a way to pass the `osapi` and `oapi` prefixes into the resolver.  I didn't see an existing nice way to do that.